cmd/root.go

```diff
@@ -80,7 +80,7 @@ var rootCmd = &cobra.Command{
 		}()
 
 		fmt.Println(asciiArt)
-		rootLogger.Sugar().Infof("Version: %s", getVersion())
+		fmt.Printf("Version: %s\n", getVersion())
```

Comment:
why not use a info log? it's easier to parse

Reply:
Similar to ascii art, I want version printed irrespective of log level.